### DISCUSSION TITLE
Committed: prevent player from retrying or exiting the map

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#19848] [Ideas]
A mod that is basically the opposite of sudden death, this mod would prevent the player from retrying unless they die, It could help players to lose the bad habit of retry spamming, and could also be helpful incase you really dont want to accidentally retry in the middle of a marathon map.

**Comment by smoogipoo** [comment]
I think it'd be nice but I'm not sure this should be a "mod". It's like a modification to the game that serves no purpose as a "mod" (i.e. doesn't affect gameplay).
Would be nice if we had local-only mods or even just settings that you can click - maybe this could be a pre-game setting (in the visual settings of player loader)? A button like "this is the one" which you click and it locks you into that gameplay attempt.

**Reply by peppy** [reply]
Can understand this perspective for a dev side, but still feel like it would be best grouped into mods in terms of organisation (from a user's perspective).

Would also be cool to see it attached to replays to give more context to the play style.
